packages/lesswrong/components/votes/ThreeAxisEmojisVoteOnComment.tsx

```diff
@@ -192,19 +227,29 @@ const EmojiReactsSection: FC<{
   const {EAEmojiPalette, ForumIcon, LWTooltip} = Components;
   return (
     <>
-      {reactions.map(({emojiOption, score}) => {
+      {reactions.map(({emojiOption, anonymous, score}) => {
```

Comment:
A couple small things about mobile, not blocking:
1. I think when this wraps, there's a bit too much space between the first and second lines
2. I think it would be nice if the agree & disagree icons stayed together when the line wraps

<img width="400" alt="Screen Shot 2023-07-11 at 12 40 37 PM" src="https://github.com/ForumMagnum/ForumMagnum/assets/9057804/8880706f-5ed9-4005-8b7b-fb031e9bc377">